Saleor Checkout mention in the README file (#218)
saleor/react-storefront: README.md

};
```

### Saleor Checkout

> React Storefront is using the new [Saleor Checkout](https://github.com/saleor/saleor-checkout) for checkout + payments. The setup is as easy as:

Reviewer: ```suggestion
React Storefront uses the new [Saleor Checkout](https://github.com/saleor/saleor-checkout) for checkout and payments. The setup is as easy as:
```

> 1. Deploy Saleor Checkout and get the url

Reviewer: ```suggestion
1. Deploy Saleor Checkout
```

> 2. Set an env variable `NEXT_PUBLIC_CHECKOUT_URL` with that url

Reviewer: ```suggestion
2. Set  the `NEXT_PUBLIC_CHECKOUT_URL` environment variable with Saleor Checkout URL
```
